config/locales/en/loan_questions.en.yml

```diff
@@ -17,6 +17,18 @@ en:
     required_loan_types: "Required for which loan types?"
     text: "Title"
 
+  linked_document:
+    cell_end: End Cell
+    cell_end_hint: Z26
+    cell_start: Start Cell
+    cell_start_hint: A1
+    delete: Delete Link
+    delete_confirmation: Are you sure you want to remove this link?
+    link: "Link to file (such as a Google spreadsheet or document)"
+    linked_document: Linked Document
+    range_of_cells: Optional range of spreadsheet cells
```

Comment:
I prefer "Optional range for Google spreadsheet"—it will only work for Google spreadsheets anyway.